### PR TITLE
<SectionNotification /> - sizes and wired type

### PR DESCRIPTION
**Pull request by frontned**
### 🔦 Summary

https://docs.google.com/document/d/1QnnF_w43EkIAAn2xvCVBEJGGkMGM8udgwXY1QDYndYM/edit

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers
- [x] 👨‍🎨 UX change is approved by the Design System UX https://zeroheight.com/7sjjzhgo2/p/86d520-11-notifications/b/55f0aa/t/58e656
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_7043/

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_7043

File: packages/wix-ui-tpa/src/components/SectionNotification/SectionNotification.st.css [hunk 1]
```diff
@@ -118,3 +122,18 @@
...
+
+.root:compact .main {
+    padding-top: value(MainCompactVerticalPadding);
+    padding-bottom: value(MainCompactVerticalPadding);
+}
+
+.root:compact .buttonsWrapper {
+    margin-top: value(MainCompactVerticalPadding);
+    margin-bottom: value(MainCompactVerticalPadding);
+}
```
Comment (sivanharel) [inline packages/wix-ui-tpa/src/components/SectionNotification/SectionNotification.st.css:139]: I would check with our designer what should be defined here and will let you know.

**Review by sivanharel — CHANGES_REQUESTED** [review]
Thanks! You did a very good job!

Some comments:
1. **index.story.tsx** - You should add the size prop for the `exampleProps` object. This will add this prop with all the values for the storybook `playground` tab.
https://github.com/wix/wix-style-react/blob/83882e07c36ce82df127d1fc5c21ea63595ef762/packages/wix-ui-tpa/src/components/SectionNotification/docs/index.story.tsx#L80

2. You should add the Settings Panel tab for the storybook.
In order to do it you should do the following:
a.  **SectionNotificationWiringExample.tsx** - Create this file and add an example for the usage of the component with the type `wired`. You should also add className={root}.b. **SectionNotificationWiringExample.st.css** - Create this file and use st-mixin in order to pass the values of the variables that you will define for the settings panel in the index.story.tsx file.c.  **index.story.tsx** - Add this tab for the story page of the component. 
You can see a reference in the DatePicker component:
https://github.com/wix/wix-style-react/blob/83882e07c36ce82df127d1fc5c21ea63595ef762/packages/wix-ui-tpa/src/components/DatePicker/docs/index.story.tsx#L156

https://github.com/wix/wix-style-react/blob/master/packages/wix-ui-tpa/src/components/DatePicker/docs/DatePickerWiringExample.st.css
 https://github.com/wix/wix-style-react/blob/master/packages/wix-ui-tpa/src/components/DatePicker/docs/DatePickerWiringExample.tsx#L15 (In your case, you only need to add one simple example.)
 
Let me know if something is not clear enough :)

File: packages/wix-ui-tpa/src/components/SectionNotification/SectionNotification.visual.tsx [hunk 2]
```diff
@@ -71,6 +73,10 @@ const tests = [
     describe: 'desktop',
     its: [
       ...getTests({ description: 'Basic' }),
+      ...getTests({
+        description: 'Basic compact',
+        size: NOTIFICATION_SIZE.compact,
+      }),
```
[the remark this answers is not in the document]
Comment (frontned) [reply packages/wix-ui-tpa/src/components/SectionNotification/SectionNotification.visual.tsx:79]: I think the `getTests` function makes sure that all types are tested, bacause it maps through the types?

File: packages/wix-ui-tpa/src/components/SectionNotification/SectionNotification.st.css (hunk 1 above)
Comment (frontned) [reply packages/wix-ui-tpa/src/components/SectionNotification/SectionNotification.st.css]: Hey, I have fixed all the comments but this one.

**Review by sivanharel — COMMENTED** [review]
Awesome!
Just a small comment :)
I will send a link for our designer for review.

**Review by sivanharel — APPROVED** [review]
Great :)

**Comment by sivanharel** [comment]
The test fails, you should run `npm run wut lint:fix` and then commit and push

**Pull request merged**